gobblin-cluster/src/test/java/org/apache/gobblin/cluster/GobblinHelixJobSchedulerTest.java

```diff
@@ -264,59 +264,61 @@ private NewJobConfigArrivalEvent createJobConfigArrivalEvent(Properties properti
     return newJobConfigArrivalEvent;
   }
 
-  private void connectAndAssertWorkflowId(String expectedSuffix, NewJobConfigArrivalEvent newJobConfigArrivalEvent, HelixManager helixManager ) throws Exception {
+  private void connectAndAssertWorkflowId(String expectedSuffix, String jobName, HelixManager helixManager) throws Exception {
     helixManager.connect();
-    String workFlowId = getWorkflowID(newJobConfigArrivalEvent, helixManager);
+    String workFlowId = getWorkflowID(jobName, helixManager);
     Assert.assertNotNull(workFlowId);
     Assert.assertTrue(workFlowId.endsWith(expectedSuffix));
   }
 
-  private String getWorkflowID (NewJobConfigArrivalEvent newJobConfigArrivalEvent, HelixManager helixManager)
+  private String getWorkflowID (String jobName, HelixManager helixManager)
       throws Exception {
     // Poll helix for up to 30 seconds to fetch until a workflow with a matching job name exists in Helix and then return that workflowID
     long endTime = System.currentTimeMillis() + 30000;
     Map<String, String> workflowIdMap;
     while (System.currentTimeMillis() < endTime) {
       try{
         workflowIdMap = HelixUtils.getWorkflowIdsFromJobNames(helixManager,
-            Collections.singletonList(newJobConfigArrivalEvent.getJobName()));
+            Collections.singletonList(jobName));
       } catch(GobblinHelixUnexpectedStateException e){
         continue;
       }
-      if (workflowIdMap.containsKey(newJobConfigArrivalEvent.getJobName())) {
-        return workflowIdMap.get(newJobConfigArrivalEvent.getJobName());
+      if (workflowIdMap.containsKey(jobName)) {
+        return workflowIdMap.get(jobName);
       }
       Thread.sleep(100);
     }
     return null;
   }
 
-  private void runWorkflowTest(Instant mockedTime, String jobSuffix,
+  private void runWorkflowTest(Duration mockedTime, String jobSuffix,
```

Comment:
Does `mockedTime` still make sense? Seems like it now represents a step duration for incrementing the clock forward in time